### ISSUE TITLE
Bug in Java BLAS routines

### ISSUE THREAD

**Issue by adampauls** [luhenry/netlib#7]
Escalating [this issue](https://github.com/scalanlp/breeze/issues/823), originally found when using Breeze. Executing the offending code snippet arrives in Java11Blas.sgemmTN, so I guessing the bug is somewhere in there.

**Comment by adampauls** [comment]
Note: the example shown in the but can be slightly further minimized with 
```scala
val outputDim = 1
val inputDim = 3
```

**Comment by adampauls** [comment]
There seems to be more than one place (like https://github.com/luhenry/netlib/blob/master/blas/src/main/java/dev/ludovic/netlib/blas/Java11BLAS.java#L1833) where 
```java
 sum11 = Math.fma(a10, b01, sum11);
 sum11 = Math.fma(a10, b02, sum12);
```

appears in the `s` version but the `d` version has 

```
sum11 = Math.fma(a10, b01, sum11);
sum12 = Math.fma(a10, b02, sum12);
```

**Comment by luhenry** [comment]
@adampauls yes, seems like a bug indeed. Let me get a fix to `v2.2.1` right away

**Comment by adampauls** [comment]
Thanks for the quick fix!
